Toolset/libraries/revidelibrary.livecodescript

```diff
@@ -125,14 +125,13 @@ end revIDEShowMessageBox
 #   If pRootObject is a stack, the object list is the stack, all its cards, and all objects on each card
 #   If pRootObject is a card the list is the card and all objects on it
 #   If pRootObject is any other control, the list is pRootObject.
-function revIDEListSearchObjectTree pRootObject, pObeyDontSearch, pMarked, pIncludeSubstacks, pIgnorePasswordProtected, pScriptsOnly, @rList
+function revIDEListSearchObjectTree pRootObject, pObeyDontSearch, pMarked, pIncludeSubstacks, pIgnorePasswordProtected, pScriptsOnly, @xList
    --delete variable sAddedObjects
    
    # Put the root object into the list first. If this is not possible because to dontSearch or card marking then
    # return an error and put empty into the list. What might also happen is that one of the child objects
    # as its dontSearch set to true. In this case, we'll return a partial list.
-   if word 1 of the name of pRootObject is among the words of "field group card" and pObeyDontSearch then
-      put empty into rList
+   if pObeyDontSearch and word 1 of the name of pRootObject is among the words of "field group card" then
```

Comment:
That's a good question... I only reversed the conditions here because the boolean check would be quicker and didn't apply much other brain power to the condition ;-)

Yes I think it should check that...